### PR TITLE
Clarify the release steps

### PR DESCRIPTION
**Pull request by alallema**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -97,6 +97,34 @@ _[Read more about this](https://github.com/meilisearch/integration-guides/blob/m
...
+
+```groovy
+implementation 'com.meilisearch.sdk:meilisearch-java:X.X.X'
+```
+
+Once the changes are merged on `main`, you can publish the current draft release via the [GitHub interface](https://github.com/meilisearch/meilisearch-java/releases).
+
+A GitHub Action will be triggered and publish a new release to Maven repository.
+
+### How to Publish the Release to Maven repository manually if the CI has failed <!-- omit in TOC -->
```
Comment (curquiza) [inline CONTRIBUTING.md:124]:
```suggestion
### How to Manually Publish the Release to Maven repository  <!-- omit in TOC -->
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -97,6 +97,34 @@ _[Read more about this](https://github.com/meilisearch/integration-guides/blob/m
...
+
+⚠️ These following steps are only required if the CI has failed to release the current version in Maven repository.
```
Comment (curquiza) [inline CONTRIBUTING.md:126]:
```suggestion
⚠️ These following steps should only be applied if it's impossible to release the current version in the Maven repository via the CI. Keep in mind publishing via the CI should always be privileged.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -97,6 +97,34 @@ _[Read more about this](https://github.com/meilisearch/integration-guides/blob/m
 
 ⚠️ Before doing anything, make sure you got through the guide about [Releasing an Integration](https://github.com/meilisearch/integration-guides/blob/main/guides/integration-release.md).
 
+#### Update the version <!-- omit in TOC -->
+
```
Comment (curquiza) [inline CONTRIBUTING.md:101]:
```suggestion
```

**Review by curquiza — CHANGES_REQUESTED** [review]
This part is not really readable because of both ⚠️
Capture d’écran 2021-09-15 à 11 25 28

Could you please
- the section `Create signature credentials (first time)` passes from `####` to `###` (size of the title) and is put at the end of Release section
- put the content of the section `Sign your files and upload to Maven Repository manually ` into the section `How to Publish the Release to Maven repository manually if the CI has failed `. So the title `Sign your files and upload to Maven Repository manually` is removed

Tell me if it's not clear

**Comment by curquiza** [comment]
checked with @alallema, not need to re-organize the CONTRIBUTING as I asked in https://github.com/meilisearch/meilisearch-java/pull/210#pullrequestreview-754875537
Only one sentence should be updated :)

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -97,6 +97,32 @@ _[Read more about this](https://github.com/meilisearch/integration-guides/blob/m
...
+A GitHub Action will be triggered and publish a new release to Maven repository.
+
+### How to Manually Publish the Release to Maven repository  <!-- omit in TOC -->
+
+⚠️ These following steps should only be applied if it's impossible to release the current version in the Maven repository via the CI. Keep in mind publishing via the CI should always be privileged.
+
 #### Create signature credentials (first time) <!-- omit in TOC -->
 
 ⚠️ All these steps (create and publish a GPG key) have already been done by the Meili team and the key is shared internally. Please ask a maintainer to get the credentials if needed.
```
Comment (curquiza) [inline CONTRIBUTING.md:128]:
```suggestion
⚠️ These step (create and publish a GPG key) has already been done by the Meili team and the key is shared internally. Please ask a maintainer to get the credentials if needed.
```

**Review by curquiza — APPROVED** [review]
LGTM :)

**Comment by alallema** [comment]
bors merge

**Pull request merged**